targets/TARGET_Samsung/security_subsystem/drivers/mb_cmd_rng.c

```diff
@@ -0,0 +1,164 @@
+/**************************************************************************/
+/* Copyright (c) Samsung Electronics Co., Ltd. All rights reserved.       */
+/*                                                                        */
+/* -INSTRUCTIONS-                                                         */
```

Comment:
This is not permissive license, please review https://os.mbed.com/docs/mbed-os/latest/contributing/license.html